.github/workflows/build.yml

```
@@ -178,6 +178,214 @@ jobs:
+      - name: Generate, sign, and tamper the SignedManifest manifests
+        shell: pwsh
+        run: |
+          $dir = $env:E2E_ARTIFACTS_DIR
+          New-Item -ItemType Directory -Path (Join-Path $dir "SignedManifest")    -Force
+          New-Item -ItemType Directory -Path (Join-Path $dir "TamperedManifest") -Force
+
+          # Build the manifest JSON that will be signed.
+          # signatureVerificationMode=Disabled so the downloaded payload (unsigned ZIP)
+          # passes Authenticode verification on the client side.
+          $manifest = @"
+{
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)